apps/docs/layouts/guides/index.tsx

```diff
@@ -103,6 +107,30 @@ const Layout: FC<Props> = (props) => {
           },
         }}
       />
+      {openVideoLightbox && (
+        <div
```

Comment:
@fsansalvadore just use <Modal> component.

Reply:
Our cmdk uses the same technique.
https://github.com/supabase/supabase/blob/master/packages/ui/src/components/Command/Command.utils.tsx/#L41-L51

Reply:
as a general rule, don't think we shoud be rolling our own overlays now, should always be using ui components or directly using radix if it's a 1 off.